app/views/expenses/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render 'layouts/header', msg: 'Add New Expense', btn: 'Home' %>
+<%= render 'layouts/header', msg: 'Add New Expense', btn: '<' %>
 <%= form_with(model: @expense, url: user_expenses_path(current_user), method: :post) do |form| %>
   <div class="my-3">
     <%= form.label :name, 'What for' %>
```

Comment:
- [OPTIONAL] I would advise that you design the **_Create Expense_** button to look identical to the other buttons that you created. It would be great to position them the same way. Remember that consistency is a key factor to good design of web pages. 💻 😎  

### Your Design

![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/77106962/cb86154b-6898-48ca-825b-5194e9696d46)

### Behance Design

![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/77106962/586d26be-297f-4c3f-9b33-1c04238b3389)